docs/root/intro/arch_overview/upstream/connection_pooling.rst

```diff
@@ -86,6 +86,28 @@ back to using HTTP/2 or HTTP/1.  This path is alpha and rapidly undergoing impro
 the default behavior result in optimal latency for internet environments, so please be patient and follow along with Envoy release notes
 to stay aprised of the latest and greatest changes.
 
+.. _arch_overview_happy_eyeballs:
+
+Happy Eyeballs Support
+----------------------
+
+Envoy supports Happy Eyeballs, `RFC6555 <https://tools.ietf.org/html/rfc6555>`_,
+for upstream TCP connections, guarded by the runtime flag
+``envoy.reloadable_features.allow_multiple_dns_addresses``.
```

Comment:
I think we should call out that support is off by default but can be enabled by adding the runtime flag ...